src/server/active_tran_server.hpp

```diff
@@ -51,10 +51,11 @@ class active_tran_server
     void disconnect_page_server ();
     bool is_page_server_connected () const;
 
-    void init_log_page_broker ();
-    void finalize_log_page_broker ();
+    void init_page_brokers ();
+    void finalize_page_brokers ();
 
-    cublog::page_broker &get_log_page_broker ();
+    template <typename PageT>
+    page_broker<PageT> &get_page_broker ();
```

Comment:
It makes not sense to have one template function unless it has a generic implementation, but since there are already two explicit implementations ... make'em two functions.

Reply:
Open up to the beauty of templates :)